client.go

```diff
@@ -154,6 +155,18 @@ var (
 	ErrMarathonDown = errors.New("all the Marathon hosts are presently down")
 	// ErrTimeoutError is thrown when the operation has timed out
 	ErrTimeoutError = errors.New("the operation has timed out")
+
+	// Default http client used when creating new marathon client and no set
+	// in provided config
+	DefaultHttpClient = &http.Client{
```

Comment:
Sry I am just used to having test package separated from main package ... we can make it private in this case.